README.rst

```diff
@@ -17,16 +18,93 @@ high-throughput imaging experiment. The measurements are stored across thousands
 cytominer-database helps you organize these data into a single database backend, such as SQLite.
 
 Why cytominer-database?
------------------------
+=======================
 While tools like CellProfiler can store measurements directly in databases, it is usually infeasible to create a
 centralized database in which to store these measurements. A more scalable approach is to create a set of CSVs per
 “batch” of images, and then later merge these CSVs.
 
-cytominer-database ingest reads these CSVs, checks for errors, then ingests them into a database backend, including
+cytominer-database ingest reads these CSVs, checks for errors, then ingests
+them into a database backend, including
 SQLite, MySQL, PostgresSQL, and several other backends supported by odo.
 
 .. code-block:: sh
 
 	cytominer-database ingest source_directory sqlite:///backend.sqlite -c ingest_config.ini
 
 will ingest the CSV files nested under source_directory into a SQLite backend
+
+How to use the configuration file
+=================================
+The configuration file ingest_config.ini must be located in the source_directory
+and can be modified to specify the ingestion.
+There are three different sections.
+
+The [filenames] section
+-----------------------
+
+.. code-block::
+
+  [filenames]
+  image   = image.csv      #or: Image.csv
+  object  = object.csv     #or: Object.csv
+
+Cytominer-Database is currently limited to the following measurement file kinds:
+Cells.csv, Cytoplasm.csv, Nuclei.csv, Image.csv, Object.csv.
+The [filenames] section in the configuration file saves the correct basename of existing measurement files
+(this may be important in the case of inconsistent capitalization).
+
+The [database_engine] section
+-----------------------------
+
+.. code-block::
+
+  [ingestion_engine]
+  engine = Parquet      #or: SQLite
+
+The [database_engine] section specifies the backend. Possible key-value pairs are:
```

Comment:
```suggestion
The [database_engine] section specifies the backend.
Possible key-value pairs are:
```

Quick fix for [one-sentence-per-line text philosophy](https://news.ycombinator.com/item?id=4642395). It makes text editing much easier in regards to github diffs. (and is a quick fix!)